### PR TITLE
something-went-wrong-page craeted

### PR DESCRIPTION
**Pull request by CamiloGilFranco**
## Summary

A server error page is created to which the user will be redirected if there is an error in the RestaurantView, RestaurantList, CarouselComponent or RestaurantFilterComponent requests

### CODE REVIEW

File: src/components/RestaurantFilterComponent/RestaurantFilterComponent.jsx [hunk 1]
```diff
@@ -18,11 +18,16 @@ const RestaurantFilterComponent = () => {
 
   const language = useSelector((state) => state.languageReducer);
 
+  const navigate = useNavigate();
+
   useEffect(() => {
     axios
       .get(`${process.env.REACT_APP_API_URL}/cuisine-categories`)
```
Comment (ht1204) [inline src/components/RestaurantFilterComponent/RestaurantFilterComponent.jsx]: const URL = process.env.REACT_APP_API_URL;

File: src/components/RestaurantList/RestaurantList.jsx [hunk 2]
```diff
@@ -6,15 +6,19 @@ import { useSearchParams } from "react-router-dom";
...
 import axios from "axios";
+import { useNavigate } from "react-router-dom";
 
 const RestaurantList = () => {
-  const data = DB;
+  const navigate = useNavigate();
 
   useEffect(() => {
     axios
       .get(`${process.env.REACT_APP_API_URL}/restaurants/withcuisines/all`)
```
Comment (ht1204) [inline src/components/RestaurantList/RestaurantList.jsx]: This URL should be in a constant imported through all the project:
`const API_URL = process.env.REACT_APP_API_URL;`

**Comment by CamiloGilFranco** [comment]
the REACT_APP_API_URL environment variable was assigned to the URL variable and consumed from there

**Review by ht1204 — APPROVED** [review]
Let's approve, but consider the comment for next PR.

**Pull request merged**